.github/workflows/unittest.yml

```diff
@@ -25,5 +25,5 @@ jobs:
         sudo apt-get install python3-pyqt5
     - name: Unittest the code with coverage
       run: |
-        xvfb-run `which coverage` run -m test
-        xvfb-run `which coverage` report -m
```

Comment:
??? Why didn't we figure this out 😆

Reply:
Haha😃

I also realized now.. but it didn't make any problem as far!
I just update this to follow other projects' rule.

Reply:
Haha😃

I also realized now.. but it didn't make any problem as far!
I just update this to follow other projects' rule.

Reply:
Oh I see 👍